fury/io.py

```diff
@@ -77,6 +78,7 @@ def load_image(filename, as_vtktype=False, use_pillow=True):
             vtk_image.SetSpacing(1.0, 1.0, 1.0)
             vtk_image.SetOrigin(0.0, 0.0, 0.0)
             arr_tmp = np.flipud(image)
```

Comment:
Can you remove this line?  It is not used since you do `arr_tmp = image` in the line below (81)

Reply:
Done!